api_ui/api.yaml

```
@@ -519,6 +519,77 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
+    get:
+      summary: Endpoint to get a link
+      operationId: GetLink
+      security:
+        - basicAuth: [ ]
+      parameters:
+        - $ref: '#/components/parameters/id'
+        - $ref: '#/components/parameters/sessionID'
+      tags:
+        - Links
+      responses:
+        '200':
+          description: Link deleted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetLinkQrCodeResponse'
+        '400':
+          $ref: '#/components/responses/400'
+        '500':
+          $ref: '#/components/responses/500'
+
+
+  /v1/credentials/links/{id}/qrcode:
+    post:
+      summary: Endpoint to create a link qr code
+      operationId: CrateLinkQrCode
```

```suggestion
      operationId: CreateLinkQrCode
```
Missing "e" in Create